module-2/backend/week_two.md

```diff
@@ -5,37 +5,74 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. At a high level, what is ActiveRecord? What does it do/allow you to do?
+  It is an ORM It allows us to "talk" with the database ad send SQL commands without writing SQL commands
 2. What is a migration?
+  It allows us to create/modify a table with given attributes. 
 3. How does a table relate to a model?
+  the model is the class of the given table. it allows us to make not only methods to call on a modle but alos set up relationships to       other models. 
 4. What kind of methods are `belongs_to`, and `has_many`? (i.e. class or instance) Give an example.
+  class methods A horse can belong to a stable and a stable has many horses. 
```

Comment:
we talked about this today in class - let me know if we need to talk about it further (i think this question here is a bit unclear!)